cpp/ps0006/cpp_ps0006_solution.cpp

```diff
@@ -4,25 +4,27 @@ using namespace std;
 void Time(int Lnum, int Fnum = 0, int step = 1);
 int main(void)
 {
-
+    // running my valid test cases 2/3
     cout << "the value is:\n";
     Time(5);
     cout << "the value is:\n";
     Time(3, 1);
     cout << "the value is:\n";
     Time(-21, 6, -3);
     cout << "the value is:\n";
-    Time(10, 10, 3);
-    // running invalid test cases, have to give clear error
-    cout << "the value is:\n";
     Time(10, 10, 0);
+    
+    // running invalid test cases, have to give clear error 3/4
+    cout << "the value is:\n";
+    Time(10, 10, 3);
     cout << "the value is:\n";
     Time(100, 10, 0);
     cout << "the value is:\n";
     Time(33, 33, -4);
     cout << "the value is:\n";
     Time(-15, 11, 4);

```

Comment:
- [ ] counter gives error message for invalid value of `step`
- [x] up counter gives error message for negative value of  `step`  
- [x] down counter gives error message for positive value of  `step`
- [ ] counter gives error message to count from a number to the same number unless `step==0`